index.html

```diff
@@ -388,6 +388,31 @@
          as specified in [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.2'>section 4.3.2</a>.
         </p>
       </section>
+
+      <section id="httpDELETE">
+        <h2>HTTP DELETE</h2>
+        <p>
+         The <code>DELETE</code> method is optional per [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.5'>section
+         4.3.5</a> and this specification does not require Fedora servers to implement it.  When a Fedora server supports this method,
+         in addition to the requirements imposed on <a>LDPR</a>s within containers outlined in [[!LDP]]
+         <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'> section 5.2.5</a>, it must also follow the additional behavior outlined
+         below.
+        </p>
+        <section id="httpDELETEDepth">
+          <h3>Depth Header</h3>
+          <p>
+          A <code>Depth</code> header <code>may</code> be included in <code>DELETE</code> requests.  Implementations may choose not to
+          support all of the header's values and may define their own default behavior if a Depth header is not present.
+          </p>
+          <p>
+          The recursion of the <code>DELETE</code> must be reckoned along the <code>ldp:contains</code> axis linking contained resources.
```

Comment:
@dannylamb That's a good question. We lifted that thinking from Solid, which [just talks about the resource's "container"](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm). That seems to imply a single container, but that's not made explicit. I can think of lots of use cases for giving multiple containers a say in authZ. There's an analogy to this `DELETE` question, but I'm not sure how close it is.